templates/4a_compensate_image_data.ipynb

```diff
@@ -14,7 +14,7 @@
    "id": "237267d0",
```

Comment:
<p>If recommended is 10-20 maybe make the default 10 as well? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>it's set at 5 per run under the assumption they will be testing on multiple runs, but I can make it more clear if they're only processing one run to up the default</p>

Reply:
<p>Given both Ben and Hadeesha's confusion about what that variable is actually doing, I think it makes sense to further clarify how it works</p>